packages/backend/src/graphql/schema.graphql

```diff
@@ -65,6 +70,10 @@ type Mutation {
   createTable(input: CreateTableInput!): TableMetadata!
   updateTable(input: UpdateTableInput!): TableMetadata!
   deleteTable(input: DeleteTableInput!): Boolean!
+  # Tiles rows
+  createRow(input: CreateTableRowInput!): ID!
+  updateRow(input: UpdateTableRowInput!): ID!
```

Comment:
nit: curious as to the reason why we hand-roll optimistic updates (e.g. in #303) instead of using apollo's one

Reply:
for apollo's optimistic update, it firsts populates the data returned to what you specify and then returns whatever the server returns. for now, i dont see the need to reflow/rerender the frontend when the server returns again since it's going to be either the same response or failure.